scripts/import-tracing-runtime.sh

```diff
@@ -42,4 +42,9 @@ cd ..
 echo "Running ./scripts/update-tracing-runtime-lock.sh $SPEC_VERSION ..."
 source ./scripts/update-tracing-runtime-lock.sh $SPEC_VERSION
 
+echo "Format with cargo fmt"
```

Comment:
Not sure that you need this in theory as format should be applied on moonbeam repo if we had the check, and if we don't there's no need to format.

Reply:
the ci is always failing on for cargo fmt check, I prefer that we format before  pushing